preload.js

```diff
@@ -1,6 +1,13 @@
 // All of the Node.js APIs are available in the preload process.
 // It has the same sandbox as a Chrome extension.
 
+// Test helpers
+const test = {
+  allPassed: () => test.done(true),
+  assert: (ok) => ok || test.fail(),
+  done: (success) => require('electron').ipcRenderer.send('test-done', success),
+  fail: () => test.done(false)
```

Comment:
```suggestion
  assert: (ok) => { if (!ok) test.fail() },
  done: (success = true) => require('electron').ipcRenderer.send('test-done', success),
  fail: () => test.done(false),
```

Another super nit, I think `test.done()` reads more fluently than `test.allPassed()` at the end of the test.

Reply:
9b3322369caab7af28e58c84726c2fed8c63eb42